packit/config/package_config.py

```diff
@@ -229,6 +230,21 @@ def get_copr_build_project_value(self) -> Optional[str]:
             )
         return projects_list[0]
 
+    def validate(self):
+        check_files = {check_file.src for check_file in self.synced_files.files_to_sync}
+        check_files.add(self.specfile_path)
+        for file_path in check_files:
+            if not Path(file_path).is_file():
+                raise PackitConfigException(f"{file_path} file does not exist")
+
+        full_downstream_url = "https://src.fedoraproject.org/rpms/{0}/raw/master/f/{0}.spec".format(
+            self.downstream_package_name
+        )
+        if not requests.get(full_downstream_url):
```

Comment:
can we use head instead of get? we don't really need to waste network bandwidth since we don't care about the content